sites/default/sqlconf.php

```diff
@@ -28,3 +28,4 @@
 //////////////////////////
 //////////////////////////
 //////////////////////////
+?>
```

Comment:
Revert these changes by copying over sqlconf.php from master. Also do the same with application.config.php above.

Reply:
of course this was done unintentionally